<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterWindow</class>
 <widget class="QMainWindow" name="PlotterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Plot">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_data_file"/>
    <addaction name="actionExport_data_file"/>
    <addaction name="actionView_data"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Vertical_In"/>
    <addaction name="actionZoom_Vertical_Out"/>
    <addaction name="separator"/>
    <addaction name="actionMaximize_orizontal"/>
    <addaction name="actionMaximize_vertical"/>
    <addaction name="actionReset_Zoom"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_Undo"/>
    <addaction name="actionZoom_Redo"/>
   </widget>
   <widget class="QMenu" name="menuCursors">
    <property name="title">
     <string>Cursors</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionInfo_data"/>
   </widget>
   <widget class="QMenu" name="menuAxis">
    <property name="title">
     <string>Axis</string>
    </property>
    <addaction name="actionBottom_Left"/>
    <addaction name="actionTop_Right"/>
    <addaction name="separator"/>
    <addaction name="actionBottom"/>
    <addaction name="actionLeft"/>
    <addaction name="actionTop"/>
    <addaction name="actionRight"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuZoom"/>
   <addaction name="menuCursors"/>
   <addaction name="menuAxis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_data_file"/>
   <addaction name="actionExport_data_file"/>
  </widget>
  <action name="actionOpen_data_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/open.png</normaloff>:/png/open.png</iconset>
   </property>
   <property name="text">
    <string>Open data file </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_data_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/save.png</normaloff>:/png/save.png</iconset>
   </property>
   <property name="text">
    <string>Export data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionZoom_Undo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/undo.png</normaloff>:/png/undo.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionZoom_Redo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/redo.png</normaloff>:/png/redo.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Cursor</string>
   </property>
  </action>
  <action name="actionInfo_data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/cursorInfo.png</normaloff>:/png/cursorInfo.png</iconset>
   </property>
   <property name="text">
    <string>Data Values</string>
   </property>
   <property name="toolTip">
    <string>Visualize cursors data values</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/zoom_horizontal_in.png</normaloff>:/png/zoom_horizontal_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom horizontal In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/zoom_horizontal_out.png</normaloff>:/png/zoom_horizontal_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom horizontal Out</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
  </action>
  <action name="actionBottom_Left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/axis_bottom_left.png</normaloff>:/png/axis_bottom_left.png</iconset>
   </property>
   <property name="text">
    <string>Bottom-Left</string>
   </property>
   <property name="toolTip">
    <string>Show/hide Bottom-Left axies</string>
   </property>
  </action>
  <action name="actionTop_Right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/axis_top_right.png</normaloff>:/png/axis_top_right.png</iconset>
   </property>
   <property name="text">
    <string>Top-Right</string>
   </property>
   <property name="toolTip">
    <string>Show/hide Top-Right axies</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/resetView.png</normaloff>:/png/resetView.png</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="toolTip">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionMaximize_vertical">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/maximize_vertical.png</normaloff>:/png/maximize_vertical.png</iconset>
   </property>
   <property name="text">
    <string>Maximize vertical</string>
   </property>
  </action>
  <action name="actionMaximize_orizontal">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/maximize_horizontal.png</normaloff>:/png/maximize_horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Maximize horizontal</string>
   </property>
  </action>
  <action name="actionZoom_Vertical_In">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/zoom_vertical_in.png</normaloff>:/png/zoom_vertical_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Vertical In</string>
   </property>
  </action>
  <action name="actionZoom_Vertical_Out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/zoom_vertical_out.png</normaloff>:/png/zoom_vertical_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Vertical Out</string>
   </property>
  </action>
  <action name="actionView_data">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/eye.png</normaloff>:/png/eye.png</iconset>
   </property>
   <property name="text">
    <string>View data</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
